packages/e2e-test-utils/src/customFont.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import visitSettings from './visitSettings';
+
+export const addCustomFont = async () => {
```

Comment:
To make this util more useful for other places (e.g. when we want to add 2-3 different fonts), it would make sense to pass the font URL as a parameter.

Similarly with `removeCustomFont` below, which ideally takes a font name. Right now it just deletes the first font in the list.

Reply:
> Similarly with removeCustomFont below, which ideally takes a font name. Right now it just deletes the first font in the list.

Makes sense that said --- any ideas on selecting the right "Remove Font" button?

I think we need to make an adjustment to the aria labels for the "Remove Font" buttons --- right now they all end up with the same label vs if we include the font name in the label.  Also on the a11y end I don't think these buttons can be reached with a screen reader as they show up on hover.

Reply:
> Also on the a11y end I don't think the "Remove Font" can be reached with a screen reader as they show up on hover.

Dang, you're right. That's not great. I will file a ticket for this.

In the meantime I suppose we have to live with this.

Reply:
#10878